packages/std/src/query/query_response.rs

```diff
@@ -13,4 +13,4 @@ use serde::de::DeserializeOwned;
 /// - multi-test/cw-sdk: create a default instance and mutate the fields
 ///
 /// This trait is crate-internal and can change any time.
-pub(crate) trait QueryResponseType: Default + DeserializeOwned {}
+pub(crate) trait QueryResponseType: DeserializeOwned {}
```

Comment:
I wonder (maybe not in scope) if this should require a few other things, like Debug, PartialEq and Clone. Certainly helpful for test code and should be implemented for all those types coming from JSON.

Reply:
True